src/work-allocation-2/containers/task-list-wrapper/task-list-wrapper.component.ts

```diff
@@ -297,4 +299,13 @@ export class TaskListWrapperComponent implements OnDestroy, OnInit {
     this.doLoad();
   }
 
+  public isCurrentUserJudicial(): boolean {
+    const userInfoStr = this.sessionStorageService.getItem(this.userDetailsKey);
+    if (userInfoStr) {
+      const userInfo: UserInfo = JSON.parse(userInfoStr);
+      const isJudge = userInfo.roles.some(role => ListConstants.JUDGE_ROLES.includes(role));
+      return isJudge;
+    }
+    return false
+  }
```

Comment:
could this use the existing AppUtils.isLegalOpsOrJudicial(roles).

Reply:
No, that seems to be different one

Reply:
Done, changed